docs/production-workflows.md

```diff
@@ -6,11 +6,12 @@ There are three main workflows for production:
 
 ### Creating a Bundle
 
+***Important**: The module names such as `app/main` used in the examples below should only be `/` separated (On Windows, do NOT use `\` as your path separator for this argument). The module names are specified in URL space; in particular, they are not file-paths.*
+
 ```
   jspm bundle app/main build.js
 ```
-
-Creates a file `build.js` containing `app/main` and all its dependencies referenced in config.js.
+Creates a file `build.js` containing `app/main` and all its dependencies referenced in config.js. \
```

Comment:
Why the trailing `/`?

Reply:
To force a newline! I guess what happens is that in my editor, md renders differently from github (despite being set for gfm). It should be trivial enough for you to fix :)